client/js/activatePlayer.js

```diff
@@ -0,0 +1,16 @@
+// Activating the next player
+function activatePlayer(game){
+    const playerDiv = document.querySelectorAll('.PlayerDiv')
+    playerDiv.forEach((player) => {
+        const playerName = player.getElementsByTagName('h1')[0].textContent // Variable to store name of the current player
+        if (playerName === 'Player ' + game.turn) { // Checking if it is the current player or not  
```

Comment:
Please improve this condition, as later on, playerName will not be "Player 1", "Player 2", etc.